BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.kt

```diff
@@ -232,7 +236,29 @@ open class BraintreeClient @VisibleForTesting internal constructor(
             if (authorization != null) {
                 getConfiguration { configuration, configError ->
                     if (configuration != null) {
-                        httpClient.get(url, configuration, authorization, responseCallback)
+                        httpClient.get(url, configuration, authorization) { response, httpError ->
+                            response?.let {
+                                try {
+                                    val params = AnalyticsEventParams(
+                                        startTime = it.startTime,
+                                        endTime = it.endTime,
+                                        endpoint = url
```

Comment:
Are we stripping out the `merchants/merchant_id/client_api` like on iOS ([here](https://github.com/braintree/braintree_ios/blob/dbe62b831090eb57ffc909634643bb30df11fb3f/Sources/BraintreeCore/BTAPIClient.swift#L428))

Reply:
Nice catch, added here: [ea509f1](https://github.com/braintree/braintree_android/pull/1032/commits/ea509f1a5a548753e744baf415ddc94aa3973605)